### DISCUSSION TITLE
Saving the output from terminal of ssh

### DISCUSSION THREAD

**Discussion by rpulikool** [Eugeny/tabby#3139] [General]
First of all i want to thank you, its really a wonderful tool, thanks for creating it!!

Its a request if you can add the feature to allow save the output of all the command when you login to a host automatically & give the option for storing it in a specific location, instead of selecting it all the time

**Comment by rpulikool** [comment]
I can see the save output plugin allows that feature but we need to select it, appreciate if you can add a feature that will get the option to store it at one place

**Reply by tritamk14** [reply]
yes, it's really helpful feature!

**Comment by Eugeny** [comment]
This is now available as an option in the latest `save-output` plugin release

**Reply by rpulikool** [reply]
Thanks i see the option but it is not saving the output automatically when i start the ssh session. Everytime i have to select save it with date and time, can you please add that option too

**Comment by 4mig4** [comment]
I tried to the save-output plugin but it does not seem to work
